packages/mui-base/src/SwitchUnstyled/SwitchUnstyled.types.ts

```diff
@@ -1,8 +1,10 @@
+import * as React from 'react';
 import { OverrideProps, Simplify } from '@mui/types';
 import { SlotComponentProps } from '../utils';
 import { UseSwitchInputSlotProps, UseSwitchParameters } from './useSwitch.types';
 
-export interface SwitchUnstyledComponentsPropsOverrides {}
+export interface SwitchUnstyledSlotPropsOverrides
+  extends Partial<Record<'root' | 'thumb' | 'input' | 'track', undefined>> {}
```

Comment:
```suggestion
  extends Partial<Record<'root' | 'thumb' | 'input' | 'track', {}>> {}
```

Isn't this the correct default value? Not sure if it will make some difference.

Reply:
It doesn't really matter from what I experienced.